library/Icinga/Application/Platform.php

```diff
@@ -59,6 +59,16 @@ public static function isLinux()
         return strtoupper(substr(self::getOperatingSystemName(), 0, 5)) === 'LINUX';
     }
 
+    /**
+     * Test of Docker
+     *
+     * @return bool
+     */
+    public static function isDocker(): bool
+    {
+        return self::isLinux() && file_exists('/.dockerenv');
```

Comment:
Didn't we talk about env vars? This also returns bool in case of any docker env (including ours) and I don't want these docker related setup changes in my env.

Reply:
Oh. Why not?

Reply:
I'm a developer. I need to test things. If I want to test the setup, I don't want to touch a system file. Additionally, the user and group stuff isn't always the same in every docker setup. **Just in the docker-icingaweb2 image**, that's why.